build.gradle

```diff
@@ -80,17 +80,18 @@ task configureGroovyAll {
         }
 
         def mergeFiles = { String relative, List<File> files ->
+            def concatenatedFiles = files*.text.join('\n')
```

Comment:
This means that we concatenate even when we don't have to. Move it to a method instead? I think the original was fine, too.

Reply:
I think all are fine, because the speed difference is not measurable. Moved it to a method